src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,123 @@
+package ui;
+
+import task.TaskList;
+import task.Task;
+import storage.Storage;
+import parser.Parser;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Prompts users for input and also to output responses to them.
+ */
+public class Ui {
+
+    private static Scanner input = new Scanner(System.in);
+
+    /**
+     * Greets the user when the chatbot just starts and ask them for an input.
+     */
+    public static void greet() {
+
+        System.out.println("Hello! I'm Tako! What can I do for you?\n");
+
+        promptInput();
+    }
+
+    /**
+     * Says goodbye to the user
```

Comment:
Minor nitpick - maybe add a period after the sentence for consistency with other javadoc method headers?